parachain/primitives/router/src/lib.rs

```diff
@@ -110,3 +178,129 @@ where
 		}
 	}
 }
+
+pub struct EthereumSovereignAccountConvertsFor<AccountId>(PhantomData<AccountId>);
+impl<AccountId: From<[u8; 32]> + Clone> Convert<MultiLocation, AccountId>
```

Comment:
actual polkadot master does not have `xcm_executor::traits::Convert` but was change to `xcm_executor::traits::ConvertLocation`, so better update deps

Reply:
Resolved since merge.